src/modules/core/components/ProgressBar/ProgressBar.tsx

```diff
@@ -1,8 +1,15 @@
 import React from 'react';
 import { defineMessages, useIntl } from 'react-intl';
 
+import { getMainClasses } from '~utils/css';
+
 import styles from './ProgressBar.css';
 
+interface Appearance {
+  theme?: 'default' | 'dark';
```

Comment:
So the 'default' theme, when in 'normal' size, has a background wrapper:

![Screenshot from 2021-03-12 11-41-19](https://user-images.githubusercontent.com/1193222/110922145-032ad000-8328-11eb-8b47-41be839aca5c.png)

While, in the 'small' size, it does not

![Screenshot from 2021-03-12 11-41-14](https://user-images.githubusercontent.com/1193222/110922173-0d4cce80-8328-11eb-9145-62e9e4285aed.png)

It think we should remove it for the 'normal' size as well to keep it consistent (an just have the grey "fill" portion of the bar)

Reply:
Sounds good to me! It also looks better without the wrapper